packages/cli/e2e/__tests__/env/env.upate.spec.ts

```diff
@@ -0,0 +1,46 @@
+// create test for checkly env add
```

Comment:
The filename of the test has a typo (missing `d` in `upate`. Also, the first line has a wrong comment.
```suggestion
// create test for checkly env update
```